apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/MoreActions/components/AssessmentSummary.tsx

```diff
@@ -24,6 +24,102 @@ type SkillAssessmentCalculated = {
+  const intl = useIntl();
+
+  return (
+    <tr
+      data-h2-border-bottom="base(3px solid black.20)"
+      data-h2-margin-bottom="base(x.5)"
+    >
+      <th
+        scope="col"
+        data-h2-padding="base(x.25 0 x.25 x1)"
+        data-h2-display="base(flex)"
+      >
+        {tableTitle}
+      </th>
+      <th scope="col">
+        <CheckIcon
+          data-h2-width="base(x1)"
+          data-h2-display="base(flex)"
+          data-h2-vertical-align="base(bottom)"
+          data-h2-margin="base(0, x.5, 0, 0)"
+          data-h2-padding="base(x.25 0)"
+          data-h2-color="base(success)"
+          aria-label={intl.formatMessage({
+            defaultMessage: "Demonstrated",
+            id: "5wKh/o",
+            description:
+              "Option for assessment decision when candidate has successful assessment.",
+          })}
+        />
+      </th>
+      <th scope="col">
+        <XCircleIcon
+          data-h2-width="base(x1)"
+          data-h2-display="base(flex)"
+          data-h2-vertical-align="base(bottom)"
+          data-h2-margin="base(0, x.5, 0, 0)"
+          data-h2-padding="base(x.25 0)"
+          data-h2-color="base(error)"
+          aria-label={intl.formatMessage({
+            defaultMessage: "Not demonstrated (Remove from process)",
+            id: "zXkLL2",
+            description:
+              "Option for assessment decision when candidate has unsuccessful assessment and been removed from the process.",
+          })}
+        />
+      </th>
+      <th scope="col">
+        <PauseIcon
+          data-h2-width="base(x1)"
+          data-h2-display="base(flex)"
+          data-h2-vertical-align="base(bottom)"
+          data-h2-margin="base(0, x.5, 0, 0)"
+          data-h2-padding="base(x.25 0)"
+          data-h2-color="base(warning)"
+          aria-label={intl.formatMessage({
+            defaultMessage: "Not demonstrated (Hold for further assessment)",
+            id: "MMtY88",
+            description:
+              "Option for assessment decision when candidate has unsuccessful assessment but on hold.",
+          })}
+        />
+      </th>
+    </tr>
+  );
+};
+
+type TableRow = {
+  name: string;
+  results: SkillAssessmentCalculated;
+};
+
+const TableBody = ({ data }: { data: TableRow[] }): JSX.Element => {
+  const generateRowMarkup = (row: TableRow): JSX.Element => {
+    return (
+      <tr key={row.name}>
+        <td data-h2-padding="base(x.25 0 x.25 x1)">{row.name}</td>
+        <td data-h2-padding="base(x.25 0 x.25 x.25)">
+          {row.results.successful}
+        </td>
+        <td data-h2-padding="base(x.25 0 x.25 x.25)">
+          {row.results.unsuccessful}
+        </td>
+        <td data-h2-padding="base(x.25 0 x.25 x.25)">{row.results.hold}</td>
+      </tr>
+    );
+  };
+
+  const criteriaRows = data.map((row) => {
+    return generateRowMarkup(row);
+  });
+
+  return <tbody>{criteriaRows}</tbody>;
+};
```

Comment:
Id just update this to:

```tsx
return (
  <tbody>
    {data.map((row) => (
      <tr key={row.name}>
        <td data-h2-padding="base(x.25 0 x.25 x1)">{row.name}</td>
        <td data-h2-padding="base(x.25 0 x.25 x.25)">
          {row.results.successful}
        </td>
        <td data-h2-padding="base(x.25 0 x.25 x.25)">
          {row.results.unsuccessful}
        </td>
        <td data-h2-padding="base(x.25 0 x.25 x.25)">{row.results.hold}</td>
      </tr>
    ))
  </tbody>
);
```

Reply:
Condensed in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8978/commits/cb7452332088efdea19a8c384dcefbd5c72f490f